manifests/d/DuckStudio/GitHubLabelsManager/1.9/DuckStudio.GitHubLabelsManager.locale.en-US.yaml

```
@@ -0,0 +1,38 @@
+# Created with Sundry.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.10.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubLabelsManager
+PackageVersion: "1.9"
+PackageLocale: en-US
+Publisher: DuckStudio
+PublisherUrl: https://duckduckstudio.github.io/yazicbs.github.io/
+PublisherSupportUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/issues
+Author: 鸭鸭「カモ」
+PackageName: GitHub Labels Manager
+PackageUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/LICENSE
+Copyright: Copyright (c) DuckStudio 鸭鸭「カモ」
+CopyrightUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/LICENSE
+ShortDescription: A GitHub Labels Manager
+Description: Tools that automatically help you copy repo labels, get repo labels, and empty existing labels.
+Tags:
+  - devtools
+  - github
+  - github-labels
+  - labels
+ReleaseNotes: |-
+  Optimization
+  - pref: output on successful optimization by @DuckDuckStudio in #29
+  Miscellaneous
+  - build(deps): bump keyring from 25.5.0 to 25.6.0 by @dependabot in #28
+  Fix
+  - fix(token). Support Fine-grained personal access tokens by @DuckDuckStudio
+
+  Full changelog: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/compare/v1.8...v1.9
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/tag/v1.9
+Documentations:
+  - DocumentLabel: README
+    DocumentUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/other-languages/en_US/README.md
```

```suggestion
    DocumentUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/other-languages/en-US/README.md
```

Honestly, I forget when I changed the folder name. 😅